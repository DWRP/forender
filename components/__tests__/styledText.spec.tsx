import * as React from "react";
import renderer from "react-test-renderer";

import { MonoText, Title } from "../StyledText"; // Substitua 'NomeDoArquivo' pelo nome do arquivo onde as funções estão definidas

describe("Teste das funções MonoText e Title", () => {
  it(`MonoText renderizado corretamente`, () => {
    const tree = renderer
      .create(<MonoText>Snapshot test for MonoText!</MonoText>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  }),
    it(`Title renderizado corretamente`, () => {
      const tree = renderer
        .create(<Title>Snapshot test for Title!</Title>)
        .toJSON();

      expect(tree).toMatchSnapshot();
    });
});
