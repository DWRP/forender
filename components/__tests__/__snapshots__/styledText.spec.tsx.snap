// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teste das funções MonoText e Title MonoText renderizado corretamente 1`] = `
<Text
  style={
    [
      {
        "color": "#000",
      },
      [
        undefined,
        {
          "fontFamily": "SpaceMono",
        },
      ],
    ]
  }
>
  Snapshot test for MonoText!
</Text>
`;

exports[`Teste das funções MonoText e Title Title renderizado corretamente 1`] = `
<Text
  style={
    [
      {
        "color": "#000",
      },
      [
        undefined,
        {
          "fontSize": 24,
          "fontWeight": "bold",
        },
      ],
    ]
  }
>
  Snapshot test for Title!
</Text>
`;
